catalog/deepfakes-influencing-turkish-election.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Deepfakes may have been used to misrepresent candidates in the Turkish election and influence voters
+companies: []
+categories: ["Deepfakes", "Misrepresentation", "Mis and Disinformation"]
+demonstrative: false
+modalities: ["Video Synthesis"]
+date: 2023-05-15
+models: []
+sources:
+  - "https://fortune.com/2023/05/15/turkeys-deepfake-influenced-election-spells-trouble/"
+---
+
+A key candidate opposing incumbent President Erdoğan, Muharrem İnce, withdrew from the
+presidential race a few days before the election due to a sex tape depicting him
+circulating on social media. İnce claimed that the video was a deepfake.
+
+The primary candidate opposing Erdoğan, Kemal Kılıçdaroğlu, was also allegedly impacted
+by deepfakes. At a rally, Erdoğan showed supporters a Kılıçdaroğlu campaign video
+exhibiting members of Kurdistan Workers' Party (PKK) singing the opposition party's
+song. PKK is designated as a terrorist organization by Turkey. The fact-checking site
+Teyit claims this video was doctored.
+
+Kılıçdaroğlu believes Russia may be behind much of the disinformation impacting the
```

Comment:
Given that this is just alleged and there isn’t concrete evidence to back it, is it worth including? I think our focus should be on the AI harm, and less on Russia itself.

Otherwise, good to merge.